### PR TITLE
add summary function and its test

### PR DESCRIPTION
**Pull request by lori94**

### CODE REVIEW

File: R/explore_summary.R [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+library("testthat")
+library("matrixStats")
+explore_summary <- function(df) {
+  # retrieve only numeric variables
+  num_index <- data.frame("index"=which(sapply(colnames(df), function(x) is.numeric(df[, x]))))
+  index <- c()
+  for(i in(1:nrow(num_index))){
+    index <- c(index, num_index[i,1])
+  }
+  num_data <- data[index]
```
Comment (fkhan72) [inline R/explore_summary.R]: I am not able to run the code. This line is creating error. Please review and modify

**Review by fkhan72 — DISMISSED** [review]
The function is not working. I have highlighted the line giving error. Please modify

File: R/explore_summary.R (hunk 1 above)
Comment (lori94) [reply R/explore_summary.R]: It runs on my device... Let me double check...

Comment (pokrovskyy) [reply R/explore_summary.R]: The problem was fixed by https://github.com/UBC-MDS/xplrrr/pull/15/commits/9552a849eaa8be1b4e55bbc0e66f418bf81220ac

**Review by pokrovskyy — APPROVED** [review]
Fixes were made, code works well now, thanks!

**Comment by lori94** [comment]
fixed typo

**Pull request merged**